cypress.json

```diff
@@ -2,5 +2,16 @@
   "baseUrl": "http://localhost:3000",
   "video": false,
   "viewportWidth": 1536,
-  "viewportHeight": 864
+  "viewportHeight": 864,
+  "integrationFolder": "cypress/integration",
+  "testFiles": [
+    "landing-page_spec.js",
```

Comment:
This was added so we can control the order the tests run in. Allowing us to run non-web3 tests while some transfers are processing to setup the account.